substrate/frame/staking/src/pallet/impls.rs

```diff
@@ -1984,6 +2100,7 @@ impl<T: Config> Pallet<T> {
 	/// * Staking ledger and bond are not corrupted.
 	fn check_ledgers() -> Result<(), TryRuntimeError> {
```

Comment:
Are there any checks we could add for the virtual bonds? e.g.:
- Payee of virtual bonds should always be `Stash(accountId)` where `accountId != who`
- Staking lock of virtual bond account == 0
- others?

Reply:
Added checks 👍